src/componant/Films.js

```diff
@@ -1,10 +1,48 @@
-import React from 'react';
+import React,{useState,useEffect} from 'react';
+import axios from 'axios'
 import './Films.css'
 
-export default function Films(){
+export default function Films(){ 
+
+    const [searchFilm,setSearchFilm]=useState('');
+    const [searchFilmValue,setSearchFilmValue]=useState([]);
+    const [error,setError]=useState(null);
+
+    useEffect(()=>{
+        if(searchFilm){
+            axios.get(`https://api.themoviedb.org/3/search/movie?api_key=304ac5e762f0abd7fc2c27e96bd42840&query=${searchFilm}
+            `)
+            .then(({data})=>{
+                if(data.total_results){
+                    setSearchFilmValue(data.results)
+                }else{
+                    setSearchFilmValue([])
+                }
```

Comment:
It not necessary to add else here, or you can show a message "No data " instead of updating  state in this case
```suggestion
                }
```